devops/scripts/load_env.sh

```diff
@@ -40,5 +40,9 @@ else
   done < <(grep -v -e '^[[:space:]]*$' -e '^#' "$1" ) # feed in via Process Substition to avoid bash subshell (http://mywiki.wooledge.org/ProcessSubstitution)
 fi
 
+if [ -n "${TRE_ID:-}" ] && [ -n "${LOCATION:-}" ]; then
```

Comment:
Don't we always use load_and_validate_env before lod_env which makes TRE_URL always available?
Should you maybe have a condition on TRE_URL too?

Reply:
As a result of our discussion - I removed this from load_env.sh and left it only in load_and_validate.sh which is being called each time before this will be called.